app/assets/src/components/PhyloTree.jsx

```diff
@@ -0,0 +1,76 @@
+import axios from "axios";
+import React from "react";
+import { Button } from "semantic-ui-react";
+import PhyloTreeViz from "./PhyloTreeViz";
+
+class PhyloTree extends React.Component {
```

Comment:
Can you do a little refactoring for future compliance:
1) Create a folder `views/phylo_tree`
2) Move all phylo tree js files in there
3) Renave `PhyloTree` -> `PhyloTreeView`